gr-blocks/lib/nlog10_ff_impl.cc

```diff
@@ -51,11 +55,18 @@ namespace gr {
       const float *in = (const float *) input_items[0];
       float *out = (float *) output_items[0];
       int noi = noutput_items * d_vlen;
-      float n = d_n;
-      float k = d_k;
 
-      for (int i = 0; i < noi; i++)
-	out[i] = n * log10(std::max(in[i], (float) 1e-18)) + k;
+      for (int i = 0; i < noi; i++) {
+          out[i] = in[i] + 1e-18;
```

Comment:
Hm, so this does change behaviour, though, I'm just not sure if it's significant. If you did
```cpp
out[i] = std::max<float>(in[i], 1e-18);
```
...you'd retain the behaviour, but obv that would incur a new performance penalty. I would assume it's still faster than the current behaviour.

Reply:
I think the loss of accuracy is fine. We handle noisy data usually, after all. However, I would like it if we mention this in the documentation for this block.

Reply:
Final version has the `std::max()` in it.